src/components/JoinUs/Learning.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { Padding, Margin } from 'styled-components-spacing'
+import styled from 'styled-components'
+import ReactMarkdown from 'react-markdown'
+import { Row, Col, CompensatedRow, CompensatedCol } from '../grid'
+import Image from '../Common/Image'
+import { H2, Paragraph } from '../Typography'
+
+import {
+  Section,
+  FirstColumn,
+  SecondColum,
+  Separator,
+  List,
+  LightH3,
+  Hr
+} from './elements'
+
+const InsightAnchor = styled.a`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+`
+
+const Learning = ({ data: { title, text, featuredInsights, list } }) => (
+  <Section greyBg>
+    <Row>
+      <FirstColumn>
+        <H2>{title}</H2>
+      </FirstColumn>
+      <SecondColum width={[1, 1, 1, 1, 6 / 12, 5 / 12]}>
+        <List>
+          <ReactMarkdown
+            renderers={{
+              heading: props => <Paragraph noMargin bold {...props} />, // eslint-disable-line react/display-name
```

Comment:
solution for content creators to reacreate the following layout with just a heading and a paragraph:

![image](https://user-images.githubusercontent.com/3236388/51123965-12c83000-1815-11e9-8839-f4ef4d89e49b.png)